docs/source/Plugin/P052_S8.rst

```diff
@@ -60,8 +60,8 @@ Task settings
 Sensor
 ^^^^^^
 
-* **GPIO <-- TX**: TX is generally set to **GPIO 13 (D7)**.
-* **GPIO --> RX**: RX is generally set to **GPIO 15 (D8)**.
+* **GPIO ← TX**: TX is generally set to **GPIO 13 (D7)**.
+* **GPIO → RX**: RX is generally set to **GPIO 15 (D8)**.

```

Comment:
Why change of these arrows?
I don't know how to type these.
Does `<--` give issues?

Reply:
Just a test. I could revert back to the `<--`